components/Auction/Auction.module.css

```diff
@@ -1,167 +1,167 @@
-.newAuctionBtnContainer {
-  text-align: center;
-}
+/*
+INFO: using px as unit for values less than 8
+*/
 
-.newAuctionBtn {
-  font-weight: 550;
-  height: 50px;
-  font-size: 1.2rem;
-  width: 14rem;
-  border: none;
-  text-align: center;
-  padding: 2px 10px;
-  border-radius: 10px;
-  background-color: #101f92;
-  color: white;
+:root {
+  --primary-auction: #0094b6;
+  --white: #ffffff;
 }
 
-.mainContainer {
+.auction {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  margin-bottom: 100px;
+  padding: 1.5rem;
+  gap: 1rem;
 }
 
-.auctionContainer {
+.auctionButtonContainer {
+  text-align: center;
+  margin-left: auto;
+}
+
+.auctionButton {
+  max-width: 13.125rem;
+  width: 100%;
+  height: 3.125rem;
+  padding: 2px 0.625rem;
+  color: var(--white);
+  background: var(--primary-auction);
+  border: 1px solid;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 0.875rem;
+  cursor: pointer;
+}
+
+.auctionButtonContentWrapper {
   display: flex;
-  width: 70%;
-  background-color: rgb(245, 235, 235);
-  justify-content: space-around;
+  justify-content: center;
   align-items: center;
-  border: 1px solid #f4f4f4;
-  border-radius: 20px;
-  box-shadow: 0 0 15px - 7px rgba(0, 0, 0, 0.65);
-  margin-bottom: 20px;
-  padding: 50px 0;
+  gap: 4px;
 }
 
-.auctionSeller {
+.mainContainer {
   display: flex;
   flex-direction: column;
-  align-items: center;
+  gap: 2rem;
+  margin-bottom: 6.25rem;
 }
 
-.auctionSeller {
-  width: 100px;
+.autionsPageTitle {
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 2rem;
+  color: #352e5b;
 }
 
-.itemInfo {
-  display: flex;
-  justify-content: center;
+.auctionWrapper {
+  display: grid;
+  grid-template-columns: minmax(17.5rem, 40.6rem);
+  gap: 1.5rem;
 }
 
-.gemImage {
-  margin-top: 20px;
+.auctionContainer {
+  background: var(--white);
+  border: 1px solid var(--white);
+  border-radius: 6px;
+  padding: 1.875rem 2rem;
+  display: grid;
+  grid-template-columns: 5rem 1fr auto;
+  gap: 2rem;
 }
 
-.currentBidPrice {
-  font-weight: 400;
+.auctionContainer:hover {
+  box-shadow: 0px 3px 6px rgba(75, 81, 91, 0.15),
+    0px 1px 3px rgba(0, 0, 0, 0.15);
```

Comment:
can you also create the variable for rgba ?

Reply:
Check this comment: https://github.com/Real-Dev-Squad/website-crypto/pull/306#discussion_r1114160275